src/zzz_od/operation/restore_charge.py

```
@@ -84,86 +127,90 @@ def select_charge_source(self) -> OperationRoundResult:
-
-    @node_from(from_name='识别当前数量', status=SOURCE_BACKUP_CHARGE)
-    @operation_node(name='处理储蓄电量恢复')
-    def handle_backup_charge(self) -> OperationRoundResult:
-        if self.required_charge is None:
-            return self.round_success()
+            return self.round_retry('未识别到当前数量', wait=0.5)
+
+        log.info(f'{source} {current_amount}')
+
+        if self._should_probe_source_in_menu():
+            # 菜单态这里只预读可用恢复量，真正的提取留到副本里点“下一步”后再确认
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)